oci/push.bzl

```diff
@@ -46,7 +46,7 @@ def _oci_push_impl(ctx):
         --layout-relative {root} \\
         --desc {desc} \\
         --target-ref {ref} \\
-        --parent-tag {tag} \\
+        --parent-tag \"{tag}\" \\
```

Comment:
Added quotes for when the tag is empty, otherwise the CLI library thinks there is no value provided.